sql/moz-fx-data-shared-prod/ads_derived/desktop_tiles_forecast_inputs_v1/metadata.yaml

```
@@ -0,0 +1,27 @@
+friendly_name: Tiles Revenue Forecast Inputs
+description: |-
+  Monthly inputs, calculated from Fx telemetry, into the desktop tiles revenue forecast.
+  Note that the forecast methodology was officially switched over from Activity Stream sessions to visits in June 2024.
+  Also, note that PingCentre (and thus sessions telemetry) was deprecated in Feb 2024.
+  Geo markets as of 2024/07/16: 'US','DE','FR','AU','CA','IT','ES','MX','BR','IN','GB','JP'.
+owners:
+  - sbetancourt@mozilla.com
+  - cmorales@mozilla.com
+labels:
+  incremental: true
+  schedule: monthly
+  change_controlled: true
+scheduling:
+  dag_name: bqetl_ads
```

If this is only running monthly, it needs to be a separate DAG
```suggestion
  dag_name: bqetl_ads_monthly
```